### DISCUSSION TITLE
Could not find "Good first issue" for the the novice contributors.

### DISCUSSION THREAD

**Discussion by sreejithitvlb** [huggingface/candle#2607] [General]
Hello, I am starting with Candle and trying to find a "Good first issue" but none of the issues are tagged with the corresponding label. Could some one help me to find a set of new issues to start with please?

**Comment by zachcp** [comment]
Maybe a good first issue would be to take model that does not yet have a WASM example and create it?  Then you can link the example in the Readme. Expand the project and get some recognition: win-win!
